core/cas-server-core-authentication/src/main/java/org/apereo/cas/authentication/handler/support/AbstractUsernamePasswordAuthenticationHandler.java

```diff
@@ -107,4 +106,16 @@ public boolean supports(final Credential credential) {
         }
         return false;
     }
+
+    public boolean matches(final CharSequence rawPassword,final String password){
+        return this.passwordEncoder.matches(rawPassword,password);
+    }
+
+    /**
+     * Used to check if there is some passwordEncoder set so matches will be utilized
+     * @return is default passwordEncoder return true
+     */
+    public boolean isNoOpPasswordEncoder(){
```

Comment:
This really is a call that likely should go away, and must be abstracted internally.